Sources/TuistAutomation/ProjectMappers/PruneUITestsProjectMapper.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistGraph
+
+public final class PruneUITestsProjectMapper: ProjectMapping {
```

Comment:
Maybe add some docs as to "what" kind of UI tests this mapper is pruning as its not immediately obvious.

Reply:
Hmm, maybe renaming the mapper to `PruneUITestsTargetsProjectMapper` would express intent better? 

> "what" kind of UI tests this mapper is pruning

What about docs like:

```
// This mapper maks all UI Tests project's targets as "to be pruned".
public final class PruneUITestsProjectMapper: ProjectMapping {
```

I am opened to other suggestions.

Reply:
I agree with Luis here. I think it's important to capture the "intent" of this mapper. Why would a user prune the UI targets from the project. Since the goal is to "skip certain test targets", what about including `skip` in the name? For example, `SkipUITestsProjectMapper`. I think the concept of "pruning" is an internal implementation detail that I don't think should be captured in the name.